marketplace/commercetools-products/src/field.js

```diff
@@ -0,0 +1,203 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button, Spinner } from '@contentful/forma-36-react-components';
+import { ApolloProvider } from '@apollo/react-components';
+import { SortableContainer } from 'react-sortable-hoc';
+import arrayMove from 'array-move';
+
+import { getApolloClient } from './apollo.js';
+import { SortableProductCard } from './product-card.js';
+
+const SortableList = SortableContainer(function SortableList({
```

Comment:
I would suggest avoiding keeping multiple component definitions within a single file to reduce the amount of lines of code and improve readability.